scripts/setup.sh

```diff
@@ -1,15 +1,22 @@
 #!/bin/bash
 
-cd ${OKTA_HOME}/${REPO}
+echo "installing node v12.13.0"
+if setup_service node v12.13.0; then
+  echo "Installed node v12.13.0 successfully"
+else
+  echo "node v12.13.0 installation failed."
+fi
 
-# Revert the cache-min setting, since the internal cache does not apply to
-# these repos (and causes problems in lookups)
-npm config set cache-min 10
+echo "installing yarn 1.21.1"
+if setup_service yarn 1.21.1; then
+  echo "Yarn v1.21.1 installed."
+else
+  echo "Yarn installation failed!"
+fi
```

Comment:
This follows convention with other UI repositories. We can choose to rely on the latest + greatest `yarn` and `node` versions, but I'll leave that for a follow-up conversation.